plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetRecordWriter.java

```diff
@@ -50,6 +51,10 @@
             REAL_WRITER_FIELD.setAccessible(true);
             INTERNAL_WRITER_FIELD.setAccessible(true);
             FILE_WRITER_FIELD.setAccessible(true);
+
+            Field memoryManager = org.apache.parquet.hadoop.ParquetOutputFormat.class.getDeclaredField("memoryManager");
+            memoryManager.setAccessible(true);
+            memoryManager.set(null, new DisabledMemoryManager(0.95f, 1024 * 1024));
```

Comment:
the ctor args are effectively ignored, so make DisabledMemoryManager ctor argless

Reply:
done